css/css-pseudo/selection-regression-p1.html

```diff
@@ -1,33 +1,29 @@
 <!doctype html>
 <meta charset="utf-8">
-<title>CSS Pseudo-Elements Test: interactions between ::selection and text-decoration</title>
+<title>CSS Pseudo-Elements Test: regression tests for ::selection backgrounds and decorations</title>
 <link rel="help" href="https://drafts.csswg.org/css-pseudo-4/#highlight-painting">
 <link rel="help" href="https://drafts.csswg.org/css-text-decor-4/#painting">
 <link rel="help" href="https://www.w3.org/TR/CSS22/zindex.html#painting-order">
-<link rel="match" href="selection-decoration-p1-ref.html">
-<meta name="assert" value="originating underlines and overlines are never painted over text, even when some of the text is selected">
+<link rel="match" href="selection-regression-p1-ref.html">
+<meta name="assert" value="originating underlines remain under unselected parts when selecting text">
 <script src="support/selections.js"></script>
 <link rel="stylesheet" href="support/highlights.css">
 <link rel="stylesheet" href="/fonts/ahem.css">
 <style>
     /*
-        The idea behind this test is that the underline should be
-        completely obscured by both the selected and unselected text.
+        This test verifies that selecting text does not cause
+        originating underline decorations, which should always paint
+        under text, to paint over unselected parts.
```

Comment:
This could perfectly be in the `<meta name="assert">`, that's the purpose of that field.